### PR TITLE
samples: wifi: Update command output and help

### PR DESCRIPTION
**Pull request by ajayparida**
[SHEL-1208] Update help text to provide information for ps_mode and reg_domain commands in wifi shell

### CODE REVIEW

File: samples/wifi/shell/README.rst [hunk 1]
```diff
@@ -109,6 +111,14 @@ Supported CLI commands
        | <dialog_token> <flow_id>
        |
        | teardown_all - Teardown all TWT flows
+   * - reg_domian
```
Comment (sachinthegreen) [inline samples/wifi/shell/README.rst:114]: reg_domain*

File: samples/wifi/shell/README.rst [hunk 2]
```diff
@@ -94,6 +94,8 @@ Supported CLI commands
        | No argument - Prints current configuration
        | on - Turns on power save feature
        | off - Turns off power save feature
+   * - ps_mode
+     - Configure Wi-Fi power save mode legacy/wmm
```
Comment (sachinthegreen) [inline samples/wifi/shell/README.rst:98]: Can't we list the values for legacy and wmm? Also wmm should be WMM.
e.g.
- | Configure Wi-Fi power save mode
  | 0 - Legacy
  | 1 - WMM

File: samples/wifi/shell/README.rst [hunk 3]
```diff
@@ -109,6 +111,14 @@ Supported CLI commands
        | <dialog_token> <flow_id>
        |
        | teardown_all - Teardown all TWT flows
+   * - reg_domian
+     - | Set or Get Wi-Fi regulatory domain
```
Comment (sachinthegreen) [inline samples/wifi/shell/README.rst:115]: get*

**Review by sachinthegreen — CHANGES_REQUESTED** [review]

File: samples/wifi/shell/README.rst [hunk 4]
```diff
@@ -94,6 +94,10 @@ Supported CLI commands
        | No argument - Prints current configuration
        | on - Turns on power save feature
        | off - Turns off power save feature
+   * - ps_mode
+     - | Configure Wi-Fi power save mode legacy/wmm
+       | 0 - Legacy
+       | 1 - WMM
```
Comment (richabp) [inline samples/wifi/shell/README.rst:100]:
```suggestion
   * - ps_mode
     - | Configure Wi-Fi power save mode
       | 0 - Legacy
       | 1 - WMM
```

File: samples/wifi/shell/README.rst [hunk 5]
```diff
@@ -109,6 +113,14 @@ Supported CLI commands
...
        |
        | teardown_all - Teardown all TWT flows
+   * - reg_domain
+     - | Set or get Wi-Fi regulatory domain
+       |
+       | Usage: wifi reg_domain [ISO/IEC 3166-1 alpha2] [-f]
+       |
+       | -f: Force to use this regulatory hint over any other regulatory hints.
+       | Note: This may cause regulatory compliance issues, use it at your
+       | own risk.
```
Comment (richabp) [inline samples/wifi/shell/README.rst:123]:
```suggestion
       | (Note that this may cause regulatory compliance issues.)
```

**Review by richabp — CHANGES_REQUESTED** [review]

**Review by kittydepa — APPROVED** [review]

**Review by sachinthegreen — APPROVED** [review]

**Pull request merged**